test/passing/comments_in_types.ml

```diff
@@ -0,0 +1,63 @@
+type t =
+  S.s        (* looks like a constructor to ocp-indent, which indents too far *)
+type t =
+  s    (* correct, because this doesn't look like a constructor to ocp-indent *)
+type t =
+    S                         (* correctly indented a little more, because... *)
+type t =
+  | S                                   (* we leave room for the vertical bar *)
+
+
+
+(* analogous value expressions, analogous to lists, some different from now *)
+let _ =
```

Comment:
Why is this test here ?

Reply:
Originally this file comes from ocp-indent regtests (https://github.com/OCamlPro/ocp-indent/blob/rewind/tests/passing/js-type.ml). Before my changes the comments « we leave room ... » and « analogous ... » were stuck together and linked to the first list.
Maybe I should only keep the first list and remove the others.

Reply:
I think that it would be easier to understand the intent of the tests going forward if the comments were changed to indicate the expected formatting, such as indicating to which syntactic phrase each comment is intended to refer.